### PR TITLE
made the update donation amount section inline on desktop

### LINKED ISSUE #273: Fixing Footer UI on Donate
The footer on the donate site is looking good but there are still some tweaks to the UI that need to be made:

**Image for reference:**
Screen Shot 2019-09-27 at 11 31 45 AM

**List of requirements**
- [x] background colour for the footer should be #000000 instead of #111111
- [x] Increase margin/padding below 'Protect the internet ...' to 24px
- [x] Increase margin/padding below 'Join our email list ...' to 24px
- [x] Increase margin/padding below email signup form field to 16px
- [x] The sign up button should share the same row as the form field and occupy 3 columns with 16px spacing between itself and the form field
- [x] _All_ text, except for links, should be white #ffffff at full opacity
- [x] All links should be #269 B7B9FA (as well as hover states for Donate, Legal, License, Participation Guidelines and Privacy)
- Screen Shot 2019-09-27 at 11 32 32 AM
- [ ] The language picker is missing @alanmoo is that within scope of this project?

Screen Shot 2019-09-27 at 11 31 45 AM

For further reference, the footer on https://foundation.mozilla.org/en/ is exactly how the footer should be

### LINKED ISSUE #276: Fix Change Amount Button
# Changes needed:
- [ ] The change amount button was originally designed to be the full width of the change amount field. However, I'm thinking we could style it closer to how we handle the footer with the button being on the same row as the form field. The button's width should be about 30% of the whole div with 16px spacing between itself and the form field:
Screen Shot 2019-09-27 at 3 29 50 PM

### PR DESCRIPTION
**Pull request by danielfmiranda**
Closes #276 

Link to demo site: [here](https://donate-wagta-276-change-8iqpkx.herokuapp.com/en-US/card/single/?source_page_id=3&currency=usd&amount=20)

Steps to test:
1.) Visit site
2.) Click the purple "change amount" link
3.) Notice that a new input and confirm button appear, and are inline. They should appear in a 70/30 format like so: 
image

4.) Resize screen to mobile width
5.) Note that the inputs are now one on top of the other like so: 
image

6.) If they appear like the screenshots above, it is working correctly! Thank you for checking

## Checklist

**Changes in Models:**
- [x] [Are my changes backward-compatible

### CODE REVIEW

**Review by KalobTaulien — APPROVED** [review]
Looks good to me. I tested locally, and it works and responds nicely. Let's wait for Kristinas approval before merging this in.

**Comment by kristinashu** [comment]
Sorry but the review link is not longer working so I can't review it right now https://donate-wagta-276-change-bexz2v.herokuapp.com/en-US/card/single/?source_page_id=3&currency=usd&amount=20

**Comment by kristinashu** [comment]
Looked good in the demo tho!

**Comment by danielfmiranda** [comment]
Hi @kristinashu! Sorry about that! Here is a new review app [link](https://donate-wagta-276-change-8iqpkx.herokuapp.com/en-US/card/single/?source_page_id=3&currency=usd&amount=20)

Thank you!

**Review by kristinashu — APPROVED** [review]
This looks perfect!

**Pull request merged**
